CHANGELOG.md

```diff
@@ -37,3 +39,166 @@ _(none)_
 ## 0.1.0 (2021-04-20)
 * Initial scaffolding of the provider against v0.56.0 of the Yandex Cloud Terraform Provider and with community
   contributions from @aladmit and @buhta!
+
+
+
+
+
+
+
+
+
+
+
+
+            From In Progress 💻
+
+        
+      
+
+        
+          
+              Restore
+
+            
+        
+    
+  
+
+  
+    
+      
+  
+  
+    
+    
+
+  
+  
+    
+    
+      Copy card link
+
+      
+      
+      
+    Archive
+    
+
+    
+      
+        Remove from project
+
+
+
+
+
+
+
+
+
+
+
+
+Upgrade to v0.68.0 of the Yandex Terraform Provider
+
+
+
+pulumi-yandex#27
+opened by stack72
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+          1 linked pull request
+          
+    
+
+          
+    
+
+        
+      
+      
+          
+
+
+  
+      
+  
+    
+  
+    
+      
+    
+      
+        
+    
+
+      
+      
+          
+        
+          
+            
+  
+  
+
+            Loading status checks…
+          
+          
+            Failed to load status checks.
+              Retry
+
+
+
+
+
+
+
+
+
+Upgrade terraform-provider-yandex to v0.69.0
+
+
+
+pulumi-yandex#28
+opened by jkodroff
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+        Press enter to move this card
+    
+
+  
```

Comment:
something happened here but also you might look into a rebase for this file in general

Reply:
Fixed. Nice catch!